mscore/pianoroll/pianoview.cpp

```diff
@@ -604,20 +604,33 @@ void PianoView::wheelEvent(QWheelEvent* event)
 //   showPopupMenu
 //---------------------------------------------------------
 
-void PianoView::showPopupMenu(const QPoint& pos)
+void PianoView::showPopupMenu(const QPoint& posGlobal)
       {
       QMenu popup(this);
 
-      QAction*  act;
+      QAction* act;
 
-      act = new QAction(tr("Note Tweaker"));
-      connect(act, &QAction::triggered, this, &PianoView::showNoteTweaker);
+      act = new QAction(tr("Cut notes"));
+      connect(act, &QAction::triggered, this, &PianoView::cutNotes);
+      popup.addAction(act);
+
+      act = new QAction(tr("Copy notes"));
+      connect(act, &QAction::triggered, this, &PianoView::copyNotes);
+      popup.addAction(act);
+
+      act = new QAction(tr("Paste notes here"));
+      connect(act, &QAction::triggered, this, &PianoView::pasteNotesAtCursor);
       popup.addAction(act);
 
-      popup.addAction(getAction("paste"));
       popup.addAction(getAction("delete"));
 
-      popup.exec(pos);
+      popup.addSeparator();
+
+      act = new QAction(tr("Note Tweaker"));
```

Comment:
I don't really like this string, can you come up with a better one?

Reply:
I'm going to remove this completely since it is now duplicated by the new levels editor.